### DISCUSSION TITLE
通过动态链接库绕过反病毒软件Hook - Break JVM « 倾旋的博客

### DISCUSSION THREAD

**Discussion by giscus** [Rvn0xsy/rvn0xsy.github.io#4] [General]
# 通过动态链接库绕过反病毒软件Hook - Break JVM « 倾旋的博客

倾旋的博客

https://payloads.online/archivers/2022-08-11/1/

**Comment by BeichenDream** [comment]
2021年的哥斯拉版本已经集成了这些技术（包括内存加载任意EXE,内存提权，内存加载Mimikatz，内存PetitPotam）使用的JNA，而且可以自定义命令模板绕过进程链查杀
